eslint.config.mjs

```diff
@@ -1,280 +1,221 @@
-// Configs
-// import standard from "eslint-config-standard"
-// import nuxt from '@nuxt/eslint-config'
+/// <reference path="eslint-typegen.d.ts" />
 
-// Plugins
-import path from 'node:path'
-import { fileURLToPath } from 'node:url'
-import jsdoc from 'eslint-plugin-jsdoc'
-import esImport from 'eslint-plugin-import'
-import unicorn from 'eslint-plugin-unicorn'
+// @ts-check
+import { createConfigForNuxt } from '@nuxt/eslint-config/flat'
+// @ts-expect-error missing types
 import noOnlyTests from 'eslint-plugin-no-only-tests'
+import typegen from 'eslint-typegen'
+// @ts-expect-error missing types
+import perfectionist from 'eslint-plugin-perfectionist'
 
-/**
- * eslintrc compatibility
- * @see https://eslint.org/docs/latest/use/configure/migration-guide#using-eslintrc-configs-in-flat-config
- * @see https://github.com/eslint/eslintrc#usage-esm
- */
-import { FlatCompat } from '@eslint/eslintrc'
-import js from '@eslint/js'
-
-const __filename = fileURLToPath(import.meta.url)
-const __dirname = path.dirname(__filename)
-
-const compat = new FlatCompat({
-  baseDirectory: __dirname,
-  recommendedConfig: js.configs.recommended
-})
-
-// TODO: Type definition?
-export default [
-  {
-    ignores: [
-      '**/dist/**',
-      '**/.nuxt/**',
-      '**/.nuxt-*/**',
-      '**/.output/**',
-      '**/.output-*/**',
-      '**/public/**',
-      '**/node_modules/**',
-      'packages/schema/schema',
-
-      // TODO: remove when fully migrated to flat config
-      '**/*.d.mts'
-    ]
-  },
-  // standard,
-  ...compat.extends('eslint-config-standard'),
-  jsdoc.configs['flat/recommended'],
-  // nuxt,
-  ...compat.extends('@nuxt/eslint-config'),
-  esImport.configs.typescript,
-  {
-    rules: {
-      'import/export': 'off'
-    }
-  },
-  {
-    files: ['**/*.vue', '**/*.ts', '**/*.mts', '**/*.js', '**/*.cjs', '**/*.mjs'],
-    languageOptions: {
-      globals: {
-        NodeJS: 'readonly',
-        $fetch: 'readonly'
-      }
+export default createConfigForNuxt({
+  features: {
+    stylistic: {
+      // TODO: Anthony highly suggests to set this to 'always'
+      commaDangle: 'never'
```

Comment:
I would highly recommend enabling comma dangling, to have a more stable diff and would save time on working with array items.